sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -177,3 +184,11 @@ def _setup_instrumentations(configurations: Dict[str, ConfigurationValue]):
                 lib_name,
                 exc_info=ex,
             )
+
+
+def _detect_attach():
```

Comment:
Probably want to move this into the utils folder.

Reply:
The _is_attach_enabled method is part of the utils. This is a distro-specific later of adding a diagnostic log in that case. Maybe a different name? Or perhaps just including it directly in the configure_azure_monitor_function?

Reply:
Not sure what you mean. Why can this not be added in utils just because it's distro-specific/